core/identity/src/cli/appkey.rs

```diff
@@ -66,8 +66,7 @@ impl AppKeyCommand {
                 let key = bus::service(model::BUS_ID)
                     .send(create)
                     .await
-                    .map_err(anyhow::Error::msg)?
-                    .unwrap();
+                    .map_err(anyhow::Error::msg)??;
```

Comment:
If you don't add anything to error message, you can just use `?` operator, because anyhow will be automatically converted